### PR TITLE
move around the input of annotsv

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/annotsv/annotsv/main.nf [hunk 1]
```diff
@@ -8,12 +8,11 @@ process ANNOTSV_ANNOTSV {
         'quay.io/biocontainers/annotsv:3.3.4--py311hdfd78af_1' }"
 
     input:
-    tuple val(meta), path(variants), path(variants_index)
+    tuple val(meta), path(variants), path(variants_index), path(snv_vcf)
```
Comment (FriederikeHanssen) [inline modules/nf-core/annotsv/annotsv/main.nf:11]: how about naming this something like `candidate_snv`. I know we have the meta.yml and all. But `variants` and `snv` in one part is confusing naming to me 🙈

Comment (nvnieuwk) [reply modules/nf-core/annotsv/annotsv/main.nf:11]: you got a point there! I'll maybe rename the original variants to 'SV_vcf` or something

Comment (nvnieuwk) [reply modules/nf-core/annotsv/annotsv/main.nf:11]: Done!

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by nvnieuwk** [comment]
Thank you @FriederikeHanssen !

**Review by maxulysse — APPROVED** [review]

**Pull request merged**